package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "hexo-renderer-sass",
-  "version": "0.4.0",
+  "version": "0.5.0",
```

```suggestion
  "version": "0.4.0",
```
The expected version will be bumped when I run `npm version major`.